auto-update-pip-packages/README.md

```diff
@@ -2,18 +2,12 @@
 
 ## Auto-update pip packages
 
-### Install
-
-Install the cron job. As root:
+This script is triggered by Continuous Integration of OpenFisca-France and OpenFisca-Web-API repositories (see their `.travis.yml` respective files).
 
-```sh
-cp cron /etc/cron.d/openfisca-auto-update-pip-packages
-systemctl restart cron
-```
+### Install
 
 You may customize these values to adapt them to your server:
-- the full path to the `update-openfisca-virtualenv.sh` script in the cron file
-- the Bash variables in `update-openfisca-virtualenv.sh` header (`PIP_PACKAGES` and `VENV_ACTIVATE`)
+- the Bash variables in `update-openfisca-virtualenv.sh` header (`VENV_ACTIVATE`)
```

Comment:
I don't think these two lines are necessary. We already have a comment in the file.